package.json

```diff
@@ -97,16 +97,16 @@
     "react-native-google-places-autocomplete": "git+https://github.com/Expensify/react-native-google-places-autocomplete.git#e12768f1542e7982d90f6449798f0d6b7f18f192",
     "react-native-haptic-feedback": "^1.13.0",
     "react-native-image-pan-zoom": "^2.1.12",
-    "react-native-image-picker": "^5.0.2",
+    "react-native-image-picker": "^5.1.0",
     "react-native-image-size": "git+https://github.com/Expensify/react-native-image-size#6b5ab5110dc3ed554f8eafbc38d7d87c17147972",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.36",
+    "react-native-onyx": "git+https://github.com/Expensify/react-native-onyx.git#4e243460ab9ea7dde1a07cb1093cea4040d7306c",
     "react-native-pdf": "^6.6.2",
     "react-native-performance": "^4.0.0",
     "react-native-permissions": "^3.0.1",
     "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#77cc9d42c474a693755941b10ee4c2d6f50e5346",
-    "react-native-plaid-link-sdk": "^7.2.0",
-    "react-native-quick-sqlite": "^5.0.3",
+    "react-native-plaid-link-sdk": "https://github.com/Pedroor/react-native-plaid-link-sdk.git",
```

Comment:
Waiting on this PR https://github.com/plaid/react-native-plaid-link-sdk/pull/518 to be merged

Reply:
@mountiny FYI, still waiting on this.